lib/ansible/modules/files/unarchive.py

```diff
@@ -71,7 +71,8 @@
     version_added: "2.1"
   extra_opts:
     description:
-      - Specify additional options by passing in an array.
+      - Specify additional options by passing in an array.  Each space-separated part of the archive program's command-line option should be a new element\
+      of the array.
```

Comment:
```suggestion
```

Reply:
Sorry this got lost - I blame github for hiding old notifications :).

I've accepted the changes.